app/views/submitted_proposals/_outcome_location_window.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="modal fade" id="outcome-window" tabindex="-1" aria-labelledby="outcome-windowLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Outcome Location</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body text-justify" style="text-align: justify;" >
+        <div class="mb-3">
+          <div class="mb-3">
+            <label>Outcome</label>
+            <%= select_tag :outcome, options_for_select([['Approved', 'Approved'], ['Rejected', 'Rejected']]), include_blank: '', class: 'form-select w-25', data: { 'submitted-proposals-target': 'outcome' } %>
+          </div>
```

Comment:
Should also have the "Declined" option.

Reply:
@brentkearney - What's the difference between `Declined` and `Rejected` because we are having email types `reject and approved` which update `outcome` column when we send email type approve/reject .That's why I've added approved/rejected options here.

Reply:
Declined is when the organizer decides not to proceed with the proposal / event, and Rejected is when the peer review process decides that the proposal will not proceed.